_posts/2019-02-05-configurable-builds-part-1.md

```diff
@@ -5,12 +5,22 @@ authors:
  - gregestren
 ---
 
-The larger your project gets, the more likely you have to build it in different
+One of Bazel's long-term goals is to make `$ bazel build //:all` "*just work*"
+so that every target builds "the right way", for whatever platform(s) you care
+about, using only flags that are genuinely interesting to you. This two-part
+series discusses the challenges involved and steps we're taking to get there.
+
+This is a deeper dive into themes covered in Bazel's [configurability roadmap]
+(https://bazel.build/roadmaps/configuration.html).
+
+## Motivation
+
+The larger your project gets, the more likely you have to build it different
 ways.
 
 Maybe you need Android, iOS, and desktop versions of your app. Maybe you build
-C++ for different platforms. Or maybe you maintain a popular library that
-different users need different features from.
+C++ for different platforms. Or maybe you maintain a popular JavaScript library
+users only want parts of to keep their .js files small.
```

Comment:
The grammar here is a little confusing. Perhaps:
... a popular JavaScript library and users want to only use parts of it to keep their .js files small

Reply:
Does a simple addition of "that" help?